src/includes/configs/customizer-options/typography.config.php

```diff
@@ -83,12 +78,17 @@
 		'bgtfw_headings_responsive_font_size' => array(
 			'section'         => 'boldgrid_typography',
 			'priority'        => 5,
-			'output_selector' => 'h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6, .bgc-heading.bgc-page-title',
+			'output_selector' => 'h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6',
 		),
 		'bgtfw_site_title_font_size' => array(
 			'section'         => 'bgtfw_site_title',
 			'priority'        => 21,
 			'output_selector' => '.site-footer .site-title > a, .' . get_theme_mod( 'boldgrid_palette_class', 'palette-primary' ) . '.site-header .site-title > a, .' . get_theme_mod( 'boldgrid_palette_class', 'palette-primary' ) . ' .site-header .site-title > a,.' . get_theme_mod( 'boldgrid_palette_class', 'palette-primary' ) . ' .site-header .site-title > a:hover, .bgc-site-title, .bgc-site-title:hover',
 		),
+		'bgtfw_site_tagline_font_size' => array(
```

Comment:
Arrows not aligned. Looks like it is more than just this instance. I'd review the whole file for alignment since it's not too big.